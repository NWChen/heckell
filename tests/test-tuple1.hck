let p : real * real;
p = (2.1, 3.4);

let x : int * real * int * char;
x = (a);

let y : int * int set * int;
y = ();

let z : (int * int ) set * int;
