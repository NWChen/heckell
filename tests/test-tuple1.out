let p: (real * real)
p = (2.1, 3.4)
let x: (int * real * int * char)
x = a
let y: (int * (int set) * int)
y = (0, 0, (2 + (x * 4)))
let z: (((int * int) set) * int)
